rskj-core/src/main/java/co/rsk/mine/MinerUtils.java

```diff
@@ -170,24 +170,24 @@
                 }
 
                 if (!(tx instanceof RemascTransaction) && tx.getGasPriceAsInteger().compareTo(minGasPrice) < 0) {
-                    logger.warn("Rejected transaction {} because of low gas account {}, removing tx from pending state.", Hex.toHexString(tx.getHash()), txSender);
+                    logger.warn("Rejected tx={} because of low gas account {}, removing tx from pending state.", hexHash, txSender);
 
                     txsToRemove.add(tx);
                     continue;
                 }
 
                 if (!expectedNonce.equals(txNonce)) {
-                    logger.warn("Invalid nonce, expected {}, found {}, tx {}", expectedNonce.toString(), txNonce.toString(), Hex.toHexString(tx.getHash()));
+                    logger.warn("Invalid nonce, expected {}, found {}, tx={}", expectedNonce, txNonce, hexHash);
```

Comment:
here we log the `BigInteger` nonce, vs the `Hex.toHexString(nonce)` before. we should make it consistent.

Reply:
:+1: